site/content/tutorial/02-reactivity/04-updating-arrays-and-objects/text.md

```diff
@@ -31,11 +33,22 @@ function addNumber() {
 }
 ```
 
-A simple rule of thumb: the name of the updated variable must appear on the left hand side of the assignment. For example this...
+However, indirect assignments to references such as this...
 
 ```js
 const foo = obj.foo;
 foo.bar = 'baz';
 ```
 
+or 
+
+```js
+function quox(thing) {
+	thing.foo.bar = 'baz';
+}
+quox(obj);
+```
+
 ...won't trigger reactivity on `obj.foo.bar`, unless you follow it up with `obj = obj`.
+
+A simple rule of thumb: the updated variable must directly appear on the left hand side of the assignment.
```

Comment:
```suggestion
A simple rule of thumb: the name of the updated variable must directly appear on the left hand side of the assignment.
```

Reply:
I removed this to prevent ambiguity in cases like this:
```js
function quox(obj) {
	obj.foo.bar = 'baz'; // won't trigger updates on obj
}
quox(obj);
```

Are you certain this should be reverted?

Reply:
oh hmm..